client/app/bundles/course/assessment/question/scribing/components/FileUploadField.jsx

```diff
@@ -14,27 +14,23 @@ const mapProps = props => ({ ...mapError(props) });
 const propTypes = {
   field: PropTypes.string.isRequired,
   label: PropTypes.string.isRequired,
-  errorMessage: PropTypes.string.isRequired,
-  validate: PropTypes.arrayOf(PropTypes.func),
   isLoading: PropTypes.bool,
+  required: PropTypes.bool,
 };
 
 const FileUploadField = (props) => {
-  const { field, label, validate, isLoading, errorMessage } = props;
+  const { field, label, isLoading, required } = props;
```

Comment:
@nicholette sorry I didn't catch this earlier. Was reading up about how to do form validation in redux, and from some of the issues, it seems like we should let this component be concerned only with the presentation logic, and declare the business logic elsewhere ([discussion here](https://github.com/erikras/redux-form/issues/65#issuecomment-140080428)).

What you can try to do is similar to other places, an example is [the following](https://github.com/Coursemology/coursemology2/blob/master/client/app/bundles/course/lesson-plan/containers/EventFormDialog/EventForm.jsx#L38-L53). That will better structure the code.

Reply:
Ok, I’ll take a look at it